charts/localstack/templates/deployment.yaml

```diff
@@ -180,6 +183,9 @@ spec:
         persistentVolumeClaim:
           claimName: {{ .Values.persistence.existingClaim | default (include "common.names.fullname" .) }}
       {{- end }}
+      {{- if ne (len .Values.persistence.extraVolumes) 0 }}
+{{ toYaml .Values.persistence.extraVolumes | indent 8 }}
```

Comment:
You can use the a preceding dash (`{{- ...}}`) to strip leading whitespaces. This allows you to indent this line (which makes it more readable).